app/controllers/thread_controller.php

```diff
@@ -4,39 +4,46 @@ class ThreadController extends AppController
 {
     const MAX_ITEMS_PER_PAGE = 5;
     const MIN_PAGE_NUMBER = 1;
+    const ADJACENT_TO_CURRENT = 4;
 
     public function index()
     {
         $threads = Thread::getAll();
+        $users = new User();
+        $user = $users->getAll();
```

Comment:
same as my comment [here](https://github.com/dandiesta/board_exercise/pull/2/files#r24880875) and [here](https://github.com/dandiesta/board_exercise/pull/2/files#r24880965) po :smile: